docs/conversation.md

```diff
@@ -9,6 +9,12 @@
 
 * Method: `GET`
 * Endpoint: `/room`
+* Data:
+
+    field | type | Description
+    ---|---|---
+    `noStatusUpdate` | int | Whether the "online" user status of the current user should be "kept-alive" or not (default 0)
```

Comment:
what are the possible values here? Apart from  0, but what does it stand for?

Reply:
1, but yeah will add it.